### ISSUE TITLE
Unexpected token parse error

### ISSUE THREAD

**Issue by jason-m-reich** [Caphyon/clang-power-tools#1197]
I just installed clang power tools and I'm getting the bellow error when I click on "Tidy" from the Visual Studio toolbar:
```
--- CLANG TIDY STARTED ---

At line:1 char:129
+ ... \extensions\oc44pdhs.5sd\Tooling\\v1\clang-build.ps1' -proj ) -file ( ...
+                                                                 ~
Unexpected token ')' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken
 

--- CLANG TIDY FINISHED ---
```

**Comment by mariru27** [comment]
Hi @jason-m-reich,

Thank you for reporting this issue
Can you try to run tidy command on a test project and send us the full log?

Best regards,
Marina

**Comment by jason-m-reich** [comment]
@mariru27 Are you asking me to just use clang-tidy "manually" from the command line? If I do that then everything works. For example:

```
➜ clang-tidy src/Vehicle.h
48869 warnings generated.
C:\Users\jreic\source\repos\inheritance-subdirectories-cmake\src\Vehicle.h:12:8: warning: 'Simulate' overrides a member function but is not marked 'override' [clang-diagnostic-inconsistent-missing-override]
  void Simulate();
       ^
C:\Users\jreic\source\repos\inheritance-subdirectories-cmake\src/Traffic_Object.h:17:16: note: overridden virtual function is here
  virtual void Simulate() = 0;
               ^
C:\Users\jreic\source\repos\inheritance-subdirectories-cmake\src\Vehicle.h:12:8: warning: annotate this function with 'override' or (rarely) 'final' [cppcoreguidelines-explicit-virtual-functions,hicpp-use-override,modernize-use-override]
  void Simulate();
       ^
                  override
Suppressed 48865 warnings (48865 in non-user code).
Use -header-filter=.* to display errors from all non-system headers. Use -system-headers to display errors from system headers as well.
```

The problem I mentioned above seems to be with the clang power tools extension trying to "access"/use windows PowerShell and/or .ps1 scripts.

**Comment by mariru27** [comment]
@jason-m-reich,

I appreciate your cooperation, it is helping me a lot

What version of Visual Studio and Clang Power Tools are you using?

Best regards,
Marina

**Comment by Zitrax** [comment]
I am getting the exact same error in version 9.5.1.16 of clang power tools with VS 2022.

**Comment by Zitrax** [comment]
#1161 seem to be a similar issue.

**Comment by Notallthatevil** [comment]
I have had this same issue with clang power tools for the last year when using CMake based projects. A fix for this would be very helpful.

**Comment by mariru27** [comment]
Hi,

Sorry for delay answer, we are looking into it and working towards a fix.
I will keep you updated as progress is made.

Best regards,
Marina

**Comment by mariru27** [comment]
Hi @jason-m-reich, @Zitrax, @Notallthatevil,

I am pleased to introduce a fix to the Clang Power Tools extension. I am excited to share this with you, and I would really appreciate it if you could take the time to test it out [v2023.2.0.1](https://drive.google.com/file/d/19F_s3HHh7oOtQ6rU_21CoEbLb0RGg9l1/view?usp=sharing) and provide your feedback. Your input is invaluable to us, and I thank you in advance for your help.

Best regards,
Marina

**Comment by mariru27** [comment]
Hi,

A release with bugfix for this issue is available in new [v2023.2.1 of Clang Power Tools](https://clangpowertools.com/CHANGELOG.html)

Thank you for your help

Have a nice day,
Marina
